src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1426,6 +1426,34 @@ void PolicyHandler::SetPrimaryDevice(const PTString& dev_id) {
   }
 }
 
+void PolicyHandler::ResetPrimaryDevice() {
+  POLICY_LIB_CHECK_VOID();
+  PTString old_dev_id = policy_manager_->PrimaryDevice();
+  policy_manager_->ResetPrimaryDevice();
+
+  connection_handler::DeviceHandle old_device_handle;
+  ApplicationManagerImpl::instance()->connection_handler()
+      ->GetDeviceID(old_dev_id, &old_device_handle);
+
+  LOG4CXX_DEBUG(
+        logger_,
+        "Old: " << old_dev_id << "(" << old_device_handle << ")");
```

Comment:
Can you please make the comment more meaningful? I.e.: Device << old_dev_id << is no longer primary device or something similar.